library/pcs_quorum_qdevice

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see LICENSE-GPLv3.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# Apache License v2.0 (see LICENSE-APACHE2.txt or http://www.apache.org/licenses/LICENSE-2.0)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: "Olivier Pouilly (@OndrejHome)"
+module: pcs_quorum
+short_description: "wrapper module for 'pcs quorum setup/destroy/change qdevice setting'"
+description:
+  - "module for setup/destroy/change qdevice setting 'pcs' utility"
+version_added: "2.10"
+options:
+  state:
+    description:
+      - "'present' - ensure that the qdevice exists"
+      - "'absent' - ensure qdevice doesn't exist"
+    required: false
+    default: present
+    choices: ['present', 'absent']
+    type: str
+  qdevice:
+    description:
+      - qdevice name
+    required: false
+    type: str
+  algorithm:
+    description:
+      - algorithm use by the cluster
+    required: false
+    choices: ['ffsplit', 'lms']
+    default: ffsplit
+    type: str
+  allowed_qdevice_changes:
+    description:
+      - "'none' - node list must match existing cluster if cluster should be present"
+      - "'update' - allow qdevice update"
+    default: none
+    required: false
+    choices: ['none', 'update']
+    type: str
+notes:
+   - Tested on Debian 10
+   - "When adding/removing qdevice, make sure to use 'run_once=True' on a cluster node
+'''
+
+EXAMPLES = '''
+- name: Setup qdevice
+  pcs_quorum_qdevice:
+    qdevice: qdevice-name
+  run_once: True
+
+- name: Setup qdevice
+  pcs_quorum_qdevice:
+    qdevice: qdevice-name
+    algorithm: ffsplit
+  run_once: True
+
+- name: delete qdevice
+  pcs_quorum_qdevice:
+    state: absent
+  run_once: True
+
+- name: Setup qdevice
+  pcs_quorum_qdevice:
+    qdevice: qdevice-name
+    algorithm: ffsplit
+  run_once: True
+
+- name: Setup qdevice
+  pcs_quorum_qdevice:
+    qdevice: qdevice-name
+    algorithm: lms
+    allowed_qdevice_changes: update
+  run_once: True
```

Comment:
(low) would it be possible to make descriptions of tasks a bit more descriptive, for example:
- Setup qdevice with default algorithm (ffsplit)
- Setup qdevice with lms algorithm
- Delete qdevice
- Setup or modify qdevice to use lms algorith

Note I have also removed one duplicate 'Setup qdevice' with ffsplit.

Reply:
Done :)

Reply:
also I don't see any changes here. Just to be sure please let me know in which commit it was fixed.

Reply:
Sorry my bad again:
8896006

Reply:
now it looks better, Thanks!